### ISSUE TITLE
pvlist gives canonical record name for record aliases

### ISSUE THREAD

**Issue by anjohnson** [epics-base/pva2pva#14]
```
epics> dbla
ao -> anj:ao
ai -> anj:ai
```
```sh
tux% pvlist 0x8D64985A000000003143910A
anj:ao
anj:ao
anj:ai
anj:ai
```

**Comment by mdavidsaver** [comment]
Was fixed by f86d61274a9554b17128154fcdd0e3e4bd829394

**Comment by mdavidsaver** [comment]
Released with Base 7.0.2
